api/v1beta2/kyma_types.go

```diff
@@ -211,14 +219,6 @@ type PartialMeta struct {
 
 const DefaultChannel = "regular"
 
-func PartialMetaFromObject(object apimetav1.Object) PartialMeta {
```

Comment:
To reviewer:

the reason to remove this function is make the `Generation` assignment trackable by IDE, with this `Object` as general input, we lost track of when the generation get assigned [here](https://github.com/kyma-project/lifecycle-manager/pull/1694/files#diff-01b10c0b6b181efc907c8638a4f6af573038dd730f8b5130be1b7071f5fba6f2R304-R306)